### ISSUE TITLE
Select Default Value

### ISSUE THREAD

**Issue by ducopro** [robsontenorio/mary#10]
Hi Robson,

First, thanx for the work!

I was starting a new project with the TALL stack and this was exactly what I needed. However, I already came across some minor things. I will list them here. Maybe you are willing to do something with 'm.

**Select Component:**
I am missing a default value. So it is not possible to preselect one of the items.

**Layout components**
Right now there is a maximum width set in de *main* and *nav* components, meaning on bigger screens there will be always a margin to the left and right.
It would be nice to have a **fullwidth** option somewhere.

Greeting from Holland,
WIm

**Comment by robsontenorio** [comment]
@ducopro 

All components are designed to be used with `wire:model` so you can set a default value, like examples from SELECT or INPUT components. 

About layout, yes it is a good idea. I will give a shot soon.

**Comment by robsontenorio** [comment]
On latest release was added `full-width`.

Check https://mary-ui.com/docs/layout

**Comment by ducopro** [comment]
Hi Robson,

Thanx for adding it!

The other issue I had with the select component, I changed to wire:model.
The point was, I was using the new #[Locked] feature for properties, and therefore only using methods to set properties.
I changed the implemenation to using the update hook for further processing.

So yes, it is true we can accomplish anything with wire:model.

However, I am creating a package where security is a serious issue. Thatś why I am a big fan of the Locked properties of livewire 3.

Just some info to maybe keep in the back of your mind.......

All in all I am still using your components wherever I can.

Greetz,
Wim
